types/index.d.ts

```diff
@@ -1696,6 +1697,15 @@ declare namespace Knex {
     Promise?: any;
   }
 
+  type LogFn = (message: string) => void;
+
+  interface LoggerConfig {
```

Comment:
Should it be called Config if signature appears to be that of a full-blown logger?

Reply:
Hmm ... I can simply rename it to Logger ?